tests/ci/install.sh

```diff
@@ -67,14 +67,16 @@ if [ -n "${ACMS_JOB}" ]; then
     composer config minimum-stability dev
     composer config prefer-stable false
     composer update "drupal/*"
-    composer update "drupal/moderation_dashboard:2.0.x-dev" "drupal/next:1.0.x-dev"
-    composer update "acquia/cohesion*" "drupal/core-*" "drupal/core" --prefer-stable
+    composer update "drupal/next:1.0.x-dev" "drupal/acquia_search:3.1.x-dev"
```

Comment:
@deepakmishra2 Any reason we removed moderation_dashboard and added acquia_search?

Reply:
@apathak18 we are now using moderation_dashboard 3.x now and not using 2.x hence removed 2.x. Secondly we dont have any release for 3.2.x for acquia_Search and hence we need to use 3.1.x of acquia_search